### PR TITLE
add the ability to check progress for multiple required languages

### PR DESCRIPTION
**Pull request by RuslanSivalnev**
add the ability to check progress for multiple required languages

Screenshot 2023-08-04 at 13 07 02
Screenshot 2023-08-04 at 13 07 43

### CODE REVIEW

**Review by bolelamx — APPROVED** [review]

File: lib/commands/check-progress-on-branch.js [hunk 1]
```diff
@@ -12,46 +12,59 @@ async function checkProgressOnBranch() {
...
+      `Crowdin: ${COLORS.GREEN}No validation performed since it is a master branch${COLORS.RESET}`,
     );
 
     return;
   }
 
-  console.log(`Crowdin: Checking language: ${configManager.get('languageToCheck')}`);
+  const requiredLanguageList = configManager.get('languagesToCheck');
+
+  console.log(`Crowdin: Checking language: ${COLORS.CYAN}${requiredLanguageList.join(`${COLORS.RESET}, ${COLORS.CYAN}`)}${COLORS.RESET}`);
```
Comment (yuriykuzin) [inline lib/commands/check-progress-on-branch.js]:
```suggestion
  console.log(`Crowdin: Checking language(s): ${COLORS.CYAN}${requiredLanguageList.join(`${COLORS.RESET}, ${COLORS.CYAN}`)}${COLORS.RESET}`);
```

File: lib/commands/check-progress-on-branch.js [hunk 2]
```diff
@@ -12,46 +12,59 @@ async function checkProgressOnBranch() {
...
-    (dataLanguage) => dataLanguage.data.languageId === configManager.get('languageToCheck'),
-  )[0];
+  const requiredLanguageProgressList = branchProgress.data.filter(
+    (dataLanguage) => requiredLanguageList.includes(dataLanguage.data.languageId),
+  );
 
-  if (currentLanguageProgress.data.phrases.total === 0) {
-    console.log(`Crowdin: ${COLORS.GREEN}Okay, no new phrases in this branch${COLORS.WHITE}`);
+  const hasLanguagesWithProgress = requiredLanguageProgressList.some(dataLanguage => dataLanguage.data.phrases.total > 0);
+  if (!hasLanguagesWithProgress) {
```
Comment (yuriykuzin) [inline lib/commands/check-progress-on-branch.js]:
```suggestion
  const hasLanguageWithNewPhrases = requiredLanguageProgressList.some(dataLanguage => dataLanguage.data.phrases.total > 0);
  if (!hasLanguageWithNewPhrases) {
```

File: lib/commands/check-progress-on-branch.js [hunk 3]
```diff
@@ -12,46 +12,59 @@ async function checkProgressOnBranch() {
...
 
     return;
   }
 
-  if (
-    currentLanguageProgress.data.phrases.translated === currentLanguageProgress.data.phrases.total
-  ) {
-    console.log(`Crowdin: ${COLORS.GREEN}Okay, translations are ready${COLORS.WHITE}`);
+  const isRequiredPhrasesTranslated = requiredLanguageProgressList.every(dataLanguage => dataLanguage.data.phrases.translated === dataLanguage.data.phrases.total);
+  if (isRequiredPhrasesTranslated) {
```
Comment (yuriykuzin) [inline lib/commands/check-progress-on-branch.js]:
```suggestion
  const isRequiredLanguagesTranslated = requiredLanguageProgressList.every(dataLanguage => dataLanguage.data.phrases.translated === dataLanguage.data.phrases.total);
  if (isRequiredLanguagesTranslated) {
```

**Review by yuriykuzin — APPROVED** [review]
Well done, thank you!

**Pull request merged**